### ISSUE TITLE
Bug d'affichage d'un saut de page dans le PDF généré avec une section contenant des groupes

### ISSUE THREAD

**Issue by ageliinas** [MTESSDev/FRW#250]
Bonjour,

Nous avons trouvé un bug au niveau de l'affichage dans le PDF généré à la fin du formulaire RDC-025 au niveau des groupes de la première section. Il se trouve qu'à partir de l'ajout de 3 groupes (années dans notre cas), la première page du PDF généré est vide, sauf le nom de la section. Le contenu se poursuit à la deuxième page, laissant ainsi une page blanche. Le problème n'est pas présent s'il y a seulement un ou deux groupes.

Résultat de l'affichage de la première page si seulement 2 groupes, par exemple :

![image](https://github.com/MTESSDev/FRW/assets/80847091/137dc9b8-5793-40c2-b2c1-3e97051bbbae)

Résultat de l'affichage de la première page dès 3 groupes, par exemple :

![image](https://github.com/MTESSDev/FRW/assets/80847091/33afeba5-46bf-47b0-baca-ae4609288523)

Voici un exemple du PDF généré avec ce problème:
[Exemple_Bug.pdf](https://github.com/MTESSDev/FRW/files/15354217/Exemple_Bug.pdf)

Merci,
Bonne journée!

[AB#82665](https://devops700.itp.extra/700/TPM/_workitems/edit/82665)

**Comment by anisite** [comment]
Bonjour, est-ce avec votre Gabarit "custom" word, parce que si nous effectuons un fix vous n'aurez pas la correction, il s'agit de changer la config des page break dans word pour lui dire ou couper, c'est pas évident de définir un emplacement pour la coupe de page pour assurer que tout est sur une page, parfois ça donne certains résultats comme ça.

**Comment by ageliinas** [comment]
Oui, c'est un gabarit custom à cause de l'ajout de notre logo. Pour le reste, on utilise le code qui nous avait été fourni. Serait-il possible d'avoir la version du code avec le 'break' dans ce cas-ci?

**Comment by anisite** [comment]
bonjour, nous pouvons difficilement faire le test aujourd'hui mais vous pourriez essayer de changer vous même en sélectionnant cette partie et en vous rendant dans paragraphe (menu contextuel) onglet enchaînements, je crois que c'est la case éviter veuves et orphelins qu'il faut décocher mais pas certain...
<img width="627" alt="image" src="https://github.com/MTESSDev/FRW/assets/3194288/0c79095c-776f-4b5d-b4f4-7da0bb30a3cc">

**Comment by ageliinas** [comment]
Bonjour, 
Nous avons testé et ca règle en partie le problème pour la première page. Toutefois, maintenant on a une page vide à la suite à la place!
[Exemple_Bug2.pdf](https://github.com/MTESSDev/FRW/files/15403638/Exemple_Bug2.pdf)

**Comment by FujuDev** [comment]
Bonjour,

Pour résoudre ce problème, veuillez sélectionner la section illustrée sur l'image dans votre gabarit Word personnalisé, puis suivre les mêmes étapes que précédemment pour faire apparaître le menu contextuel.

Ensuite, assurez-vous que les cases soient cochées comme indiqué dans l'image ci-dessous :
![image](https://github.com/user-attachments/assets/b34781b8-be1a-4e2a-8c9b-fca1efc0e1b0)

Si vous avez d'autres questions ou si ce processus ne résout pas votre problème, n'hésitez pas à nous recontacter.

**Comment by FujuDev** [comment]
![image](https://github.com/user-attachments/assets/d5c12472-9b68-4676-9fd6-911e8048b415)
Vous pouvez aussi vérifier à cet emplacement.

**Comment by Hutson48** [comment]
@ageliinas nous allons fermer cette issue car elle semble avoir été répondu par @FujuDev. Si jamais vous éprouvez encore des problèmes, n'hésitez pas à la réouvrir.
